stl/inc/yvals_core.h

```diff
@@ -39,6 +39,7 @@
 // P0063R3 C11 Standard Library
 // P0074R0 owner_less<>
 // P0092R1 <chrono> floor(), ceil(), round(), abs()
+// P0202R3 constexpr For <algorithm> And exchange()

```

Comment:
These features are incorrectly listed in the "implemented unconditionally" section.

Reply:
There are days when I need to kick myself. This is one of those days.